po/nl.po

```diff
@@ -189,7 +189,7 @@ msgstr "Kan niet verbinden met ratbagd"
 #. Translators: "it" here is ratbagd.
 #: piper/window.py:57
 msgid "Please make sure it is running"
```

Comment:
You should update this string as well; in general running `ninja -C build piper-update-po` is the easiest.